inc/classes/class-rocket-background-database-optimization.php

```diff
@@ -59,97 +59,97 @@ protected function task( $item ) {
 
 		switch ( $item ) {
 			case 'revisions':
-				$query = $wpdb->get_col( $wpdb->prepare( "SELECT ID FROM $wpdb->posts WHERE post_type = %s", 'revision' ) );
+				$query = $wpdb->get_col( "SELECT ID FROM $wpdb->posts WHERE post_type = revision" );
```

Comment:
The `'` are missing in `WHERE post_type = 'revision'`.
Same for all next queries.